### PR TITLE
sidecar: rm uneffective version of mysql56 #203

### LINKED ISSUE #203: [bug] Not effective version of mysql56
**Describe the problem**

The mysql version will never be `mysql56`:
```
166 // buildXenonConf build a config file for xenon.
167 func (cfg *Config) buildXenonConf() []byte {
               ....
               ....
171 
172         version := "mysql80"
173         if cfg.MySQLVersion.Major == 5 {
174                 if cfg.MySQLVersion.Minor == 6 {
175                         version = "mysql56"
176                 } else {
177                         version = "mysql57"
178                 }
179         }
```
In GetMySQLVersion, no matter what mysql version is, it always be 5.7 here.
```
113 // GetMySQLVersion returns the MySQL server version.
114 func (c *Cluster) GetMySQLVersion() string {
115         version := c.Spec.MysqlVersion
116         // lookup for an alias, usually this will solve 5.7 to 5.7.x
117         if v, ok := utils.MySQLTagsToSemVer[version]; ok { 
118                 version = v
119         }       
120                 
121         if _, ok := utils.MysqlImageVersions[version]; !ok {
122                 version = utils.MySQLDefaultVersion
123         }       
124         
125         return version
126 }
```

**To Reproduce**

1.

**Expected behavior**

**Environment:**

- RadonDB MySQL version:

### PR DESCRIPTION
**Pull request by hustjieke**
[summary]
rm uneffective version of mysql56, currently we do not support mysql 5.6
or lower version

[test case]
N/A

[patch codecov]
N/A

### Which issue(s) this PR fixes?

Fixes #203

### What this PR does?

Summary:
rm uneffective version of mysql56, currently we do not support mysql 5.6
or lower version

### CODE REVIEW

File: sidecar/config.go [hunk 1]
```diff
@@ -171,11 +171,9 @@ func (cfg *Config) buildXenonConf() []byte {
 
 	version := "mysql80"
 	if cfg.MySQLVersion.Major == 5 {
-		if cfg.MySQLVersion.Minor == 6 {
-			version = "mysql56"
-		} else {
-			version = "mysql57"
-		}
+		// currently we do not support mysql 5.6 or lower version.
+		log.Info("currently we do not support mysql 5.6 or lower version, default we set mysql 5.7")
```
Comment (acekingke) [inline sidecar/config.go:175]: log.Error may be better, Info level can be filtered in release version

**Comment by hustjieke** [comment]
ping @acekingke   please. review again.

File: sidecar/config.go [hunk 2]
```diff
@@ -171,11 +172,11 @@ func (cfg *Config) buildXenonConf() []byte {
 
 	version := "mysql80"
 	if cfg.MySQLVersion.Major == 5 {
-		if cfg.MySQLVersion.Minor == 6 {
-			version = "mysql56"
-		} else {
-			version = "mysql57"
+		// currently we do not support mysql 5.6 or lower version.
```
Comment (zhyass) [inline sidecar/config.go:175]: https://github.com/radondb/radondb-mysql-kubernetes/issues/203#issuecomment-912224765

**Comment by andyli029** [comment]
Next pr.

**Pull request closed without merging**